### PR TITLE
add 1351

### PR DESCRIPTION
**Pull request by Rwatana**
Please comment on this task.

### CODE REVIEW

File: 1351_count_negative_numbers_in_a_sorted_matrix.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+from typing import List
+
+
+class Solution:
+    
+    def countNegatives(self, grid: List[List[int]]) -> int:
+        yoko = len(grid[0])
+        tate = len(grid)
```
Comment (ryujumoto) [inline 1351_count_negative_numbers_in_a_sorted_matrix.py]: Don't use Japanese name for variable name.
In this case, it's better to use "row" for "tate" and "column" for "yoko".

File: 1351_count_negative_numbers_in_a_sorted_matrix.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+        i = 0
+        j = yoko -1 
+        count = 0
+        while i < tate and j >= 0:
+            if grid[i][j] < 0:
+                count = count + 1
+
```
Comment (ryujumoto) [inline 1351_count_negative_numbers_in_a_sorted_matrix.py]: Don't insert a new line.

File: 1351_count_negative_numbers_in_a_sorted_matrix.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    def countNegatives(self, grid: List[List[int]]) -> int:
+        yoko = len(grid[0])
+        tate = len(grid)
+
+        i = 0
+        j = yoko -1 
+        count = 0
+        while i < tate and j >= 0:
+            if grid[i][j] < 0:
+                count = count + 1
```
Comment (ryujumoto) [inline 1351_count_negative_numbers_in_a_sorted_matrix.py]: count += 1

**Review by ryujumoto — COMMENTED** [review]
Check my feedback.

File: 1351_count_negative_numbers_in_a_sorted_matrix.py (hunk 1 above)
Comment (Rwatana) [reply 1351_count_negative_numbers_in_a_sorted_matrix.py]: Sure, I revised my code. Thanks for teaching.

File: 1351_count_negative_numbers_in_a_sorted_matrix.py (hunk 2 above)
Comment (Rwatana) [reply 1351_count_negative_numbers_in_a_sorted_matrix.py]: Sure, I revised my code. Thanks for teaching.

**Comment by Rwatana** [comment]
I think I checked all points. Please check my code

**Comment by ryujumoto** [comment]
It looks good. Let's merge this branch to the main one.

**Pull request merged**